stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+        name: weave-flux
+      name: weave-flux
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    apiVersion: v1
+    kind: ServiceAccount
+  - metadata:
+      labels:
+        name: weave-flux-agent
+        app: weave-flux
+        weave-cloud-component: flux
+        weave-flux-component: agent
+      name: weave-flux-agent
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    spec:
+      template:
+        metadata:
+          labels:
+            name: weave-flux-agent
+            app: weave-flux
+            weave-cloud-component: flux
+            weave-flux-component: agent
+        spec:
+          containers:
+            - name: agent
+              image: 'quay.io/weaveworks/fluxd:0.1.0'
+              imagePullPolicy: IfNotPresent
+              args:
+                - '--token={{.Values.ServiceToken}}'
+          serviceAccountName: weave-flux
+      replicas: 1
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+  - metadata:
+      labels:
+        name: weave-scope
+      name: weave-scope
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    apiVersion: v1
+    kind: ServiceAccount
+  - metadata:
+      labels:
+        name: weave-scope-agent
+        app: weave-scope
+        weave-cloud-component: scope
+        weave-scope-component: agent
+      name: weave-scope-agent
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-e733d54",
+            "original-request": {
+              "url": "/k8s.yaml?t=REDACTED",
+              "date": "Tue Mar 21 2017 11:33:13 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    spec:
+      template:
+        metadata:
+          labels:
+            name: weave-scope-agent
+            app: weave-scope
+            weave-cloud-component: scope
+            weave-scope-component: agent
+          annotations:
+            scheduler.alpha.kubernetes.io/tolerations: >-
+              [{"key":"dedicated","operator":"Equal","value":"master","effect":"NoSchedule"}]
```

Comment:
This one is standard, actually will change if we check-in 1.6 version of this manifest instead, which leads to another question – how are Kubernetes versions handled in charts repo? Our manifests look quite different between 1.5 and 1.6 (this one is 1.5).